### ISSUE TITLE
[Jaws2018 + IE11] Aria-describedby not read on the elements which have default focus on a webpage

### ISSUE THREAD

**Issue by ShardulGhusey** [FreedomScientific/VFO-standards-support#19]
### Summary
> Aria-describedby text is not read on the elements which have default focus on a webpage, it is specifically observed when the element has role="dailog"

### Expected result
>The aria-describedby text should be read when focus is on that element irrespective of how the focus is set on it.

### Actual result
> The aria-describedby text is not read on an element when it has default focus.

### Additional Information
Check the codepen url [https://gist.github.com/ShardulGhusey/22c785937882491ca609b095778de114](url)

#### JAWS version and build number
Jaws 2018
 
#### Operating System and version
Windows 10

#### Browser and version:
IE 11 -- 11.674

**Comment by TPGJRogers** [comment]
Issue is marked IE11. Closing.
